blockchain/stake/internal/tickettreap/common_test.go

```diff
@@ -38,6 +39,24 @@ func uint32ToKey(ui uint32) Key {
 	return key
 }
 
+// TestSizeValues are correct.
+func TestSizeValues(t *testing.T) {
+	var node treapNode
+	if unsafe.Sizeof(node) != nodeFieldsSize {
```

Comment:
Ah, no worries.  I thought if it was in test it might get a pass, unaware there were platform compilation issues.  I was going to suggest conditional compilation as I still think it's really valuable, but I see I can use reflect.TypeOf(node).Size() instead which is already being used in the test packages.